src/scss/abstracts/_background-mixins.scss

```diff
@@ -1,87 +1,88 @@
 @use "sass:map";
-@import 'variables';
-@import 'placeholders';
+@forward 'variables' show $imgpath;
```

Comment:
If I'm understanding the [docs](https://sass-lang.com/documentation/at-rules/forward/) on this, it seems like the purpose of this is to make it so that `$imgpath` is available from this file (`background-mixins.scss`) when it is `use`'d in another?